tests/tests_app/utilities/test_imports.py

```diff
@@ -3,7 +3,14 @@
 
 import pytest
 
-from lightning_app.utilities.imports import requires
+from lightning_app.utilities.imports import _get_extras, requires
+
+
+@mock.patch("lightning_app.utilities.imports.metadata.requires")
```

Comment:
I would suggest not mocking this for better coverage, knowing that the test might break if requirements are changed in the future. But it would be an esasy fix

Reply:
Yeah, mocking doesn't seem to be working either haha

Reply:
Done